test/should.js

```diff
@@ -545,6 +545,64 @@ describe('should', function() {
     }, "blah: expected 1 to have property 'length'");
   });
 
+  it('within(start, finish) (dates)', function(){
+    var now = new Date();
+    var oneSecondBefore = new Date(now.getTime() - 1000);
+    var oneSecondAfter = new Date(now.getTime() + 1000);
+    var nowUTC = now.toUTCString();
+    var beforeUTC = oneSecondBefore.toUTCString();
+    var afterUTC = oneSecondAfter.toUTCString();
+
+    (now).should.be.within(oneSecondBefore, oneSecondAfter);
+    (now).should.be.within(now, oneSecondAfter);
+    (now).should.be.within(now, now);
+    (oneSecondAfter).should.not.be.within(oneSecondAfter, oneSecondBefore);
+
+    err(function(){
+      (now).should.not.be.within(now, oneSecondAfter, 'blah');
+    }, "blah: expected " + nowUTC + " to not be within " + nowUTC + ".." + afterUTC);
+
+    err(function(){
+      (oneSecondBefore).should.be.within(now, oneSecondAfter, 'blah');
+    }, "blah: expected " + beforeUTC + " to be within " + nowUTC + ".." + afterUTC);
+
+    err(function(){
+      ([]).should.have.length.within(now, 100, 'blah');
+    }, "blah: the arguments to within must be numbers");
+
+    err(function(){
+      (now).should.have.lengthOf.within(50, now, 'blah');
+    }, "blah: expected " + nowUTC + " to have property 'length'");
+
+    err(function () {
+      (now).should.have.length.within(5, 7);
+    }, "expected " + nowUTC + " to have property 'length'");
+
+    err(function () {
+      (0).should.be.within(0, now, 'blah');
+    }, "blah: the arguments to within must be numbers");
+
+    err(function () {
+      (1).should.be.within(now, 1, 'blah');
+    }, "blah: the arguments to within must be numbers");
+
+    err(function () {
+      (now).should.be.within(1, now, 'blah');
+    }, "blah: the arguments to within must be dates");
+
+    err(function () {
+      (now).should.not.be.within(now, 1, 'blah');
+    }, "blah: the arguments to within must be dates");
+
+    err(function () {
+      (now).should.not.be.within(null, now, 'blah');
+    }, "blah: the arguments to within must be dates");
+
+    err(function () {
+      (now).should.not.be.within(now, null, 'blah');
+    }, "blah: the arguments to within must be dates");
+  });
+
```

Comment:
Ok, if I understood correctly, just omit parenthesis and do `now.should` instead of `(now).should`?